pipelines/prefect_flows/common_stages.py

```diff
@@ -53,15 +53,26 @@ def load_data(
 
     dist_alerts = _load_zarr(dist_zarr_uri)
 
-    country = _load_zarr(country_zarr_uri)
+    # reindex to dist alerts to avoid floating point precision issues
+    # when aligning the datasets
+    # https://github.com/pydata/xarray/issues/2217
+    country = _load_zarr(country_zarr_uri).reindex_like(
```

Comment:
Should we just move the reindex_like into the _load_zarr function then, or is there a case where you don't want to do this?